src/client-apps/nuxeo-drive/nuxeo-drive-release-notes.md

```diff
@@ -9,33 +9,43 @@ review:
 toc: true
 ---
 
-Welcome to the Release Notes for **Nuxeo Drive 5.2.6**
+Welcome to the Release Notes for **Nuxeo Drive 5.2.7**
 
-**Status**: <font color="#0066ff">**Release**</font> </br>
-<i class="fa fa-long-arrow-right" aria-hidden="true"></i> [Changelog](https://github.com/nuxeo/nuxeo-drive/blob/master/docs/changes/5.2.6.md)
+**Status**: <font color="#ff0000">**Beta**</font> </br>
+<i class="fa fa-long-arrow-right" aria-hidden="true"></i> [Changelog](https://github.com/nuxeo/nuxeo-drive/blob/master/docs/changes/5.2.7.md)
 
 ## General
 
-### Fixes
+### Improvements
 
-#### [macOS] Fix windows maximization not being reversible
+#### Better Handle Failed Migrations
 
-On macOS, the maximization of certain windows could not be undone and the user was stuck in fullscreen. The only way to get out of the situation was to use the activity monitor to force kill Drive. This issue has been partially fixed by making the user able to restore the window to it's normal size when pressing the `Escape` key.
+Thanks to this improvement, following a failed migration, the application will automatically downgraded to the previous working version and will not allow to upgrade to the problematic version anymore. But it will allow to upgrade to the next next version.
```

Comment:
```suggestion
Thanks to this improvement, following a failed migration, the application will be automatically downgraded to the previous working version and will not allow to upgrade to the problematic version anymore. But it will allow to upgrade to the next next version.
```